packages/server/utils/PubSubPromise.ts

```diff
@@ -60,10 +61,11 @@ export default class PubSubPromise<Request extends BaseRequest, Response> {
         sendToSentry(new Error('REDIS JOB ALREADY EXISTS'), {tags: {jobId}})
       }
       this.jobs[jobId] = {resolve, timeoutId}
-      const {serverChannel, ...rest} = request
-      const message = JSON.stringify({jobId, request: rest})
-      if (serverChannel) {
-        this.publisher.publish(serverChannel, message)
+      const {executorServerId, ...rest} = request
+      const message = JSON.stringify({jobId, socketServerId: SERVER_ID, request: rest})
+      if (executorServerId) {
+        const executorChannel = GQLExecutorId.join(executorServerId)
```

Comment:
-1 the result of `...Id.join(...)` should be an id, now it's the opposite, so it should technically be `split(...)`

Reply:
My issue is that we're passing around ids and then we use them by calling `GQLExecutorId.join(id)` to get a channel name. I think safest would be to call it `GQLExecutorChannel.join(id)` or if you like the `Id` suffix at least call it `GQLExecutorChannelId`. Naming is hard.

Reply:
oh! that makes sense